NOTES.md

```diff
@@ -1,3 +1,107 @@
+# Release 2.1.0-beta (Forbidden Planet)
+
+-- issues up-to-date with the 2.1 milestone as of 2015-07-06 23:00 UTC
+-- contributors " " next as of 3dac994 and docs:master as of 0b661c4
+
+Release highlight:
+
+* Automatic failover
+* More flexible administration for servers and tables
+
+Read the [blog post][2.1-beta] for more details.
+
+[2.1-beta]: http://rethinkdbblog.com/blog/2.1-beta/
+
+## Compatibility ##
+
+This beta release does not include automatic migration of data
+directories from older versions of RethinkDB. However the final 2.1.0
+version will include that feature for versions of RethinkDB down to
+1.14.0. Upgrading directly from RethinkDB 1.13 will no longer be
+supported.
+
+### API-breaking changes ###
+
+* `.split('')` now treats the input as UTF8 instead of an array of bytes.
+* `null` values in compound index are no longer discarded.
+* The new `read_mode="outdated"` optional argument replaces `use_outdated=True`
+* The format of the system tables has changed
+
+## New features ##
+
+* Server
+ * Added automatic failover and semi-lossless rebalance based on raft (#223)
+ * Backfills are now interuptible and reversible (#3886, #3885)
+ * `table.reconfigure()` now works even if some servers are disconnected (#3913)
+ * Replicas can now be marked as voting or non-voting (#3891)
+ * Added an emergency repair feature to restore table availability if consensus is lost (#3893)
+ * Reads can now be made against a majority of replicas (#3895)
+ * Added an emergency read mode that extracts data directly from a given replica for data recovery purposes (#4388)
+ * Now allow servers to go away without issues if they don't have any responsibilities (#1790)
```

Comment:
How about
> Servers with no responsibilities can now be removed from clusters without raising an issue

(the removal doesn't involve any special procedure from the server's side, you just take it down)